app/views/qsets/_question_page_header_large_screen.html.erb

```diff
@@ -9,3 +9,10 @@
     <%= render "all_mine_other_filter" %>
   </div>
 </div>
+<div class = "row" class="col-xs-12">
+  <div id="breadcrumbs">
```

Comment:
we don't need this <div id="breadcrumbs"> (also just to keep it consistent with your code in _question_page_header_small_screen.html.erb line 7-9)